plugins/alphabet-sort/src/editor.tsx

```diff
@@ -0,0 +1,248 @@
+import * as React from 'react'
+import { ContactCard } from './contact-card.component'
+import { styled } from '@serlo/editor-ui'
+import * as R from 'ramda'
+import {
+  renderIntoSidebar,
+  Input,
+  Textarea,
+  Dropdown,
+  Button,
+  ButtonGroup
+} from '@splish-me/editor-ui'
+import { Upload } from '@serlo/editor-plugin-image'
+import { PluginEditorProps } from '@splish-me/editor'
+import {
+  AlphabetSortProps,
+  ContactProps
+} from '@serlo/editor-plugin-alphabet-sort-renderer'
+
+const AddButton = styled.button({
+  outline: 'none',
+  width: '100%',
+  border: 'none',
+  marginBottom: '10px',
+  backgroundColor: '#95bc1a'
+})
+const SortContainer = styled.div({
+  display: 'flex',
+  flexWrap: 'wrap',
+  width: '100%',
+  alignSelf: 'center'
+})
+const uploadConfig = {
+  url: 'https://serlo-upload.free.beeceptor.com',
```

Comment:
You should pass the upload config to the plugin. This still has the dev stuff in it.